installation/aws/VmClarity.cfn

```diff
@@ -379,15 +328,10 @@ Resources:
             02reload_systemctl:
               command: systemctl daemon-reload
 
-            02enable_exploit_db_fetcher_timer:
-              command: systemctl enable exploit_fetcher.timer
-            03start_restart_exploit_db_fetcher_timer:
-              command: systemctl restart exploit_fetcher.timer
-
-            03enable_exploit_server:
-              command: systemctl enable exploit_server.service
-            04start_restart_exploit_server:
-              command: systemctl restart exploit_server.service
+            03enable_exploit_db_fetcher_timer:
```

Comment:
maybe the `_fetcher_timer` is not needed?

Reply:
ah the name here is just wrong